README.md

```diff
@@ -4,6 +4,16 @@
 
 # Cylc UI
 
+## TODO
+
+I haven't managed to configure webpack to pass through the WASM files yet
+so for the moment you have to manually copy the wasm file over into dist
+the first time you build.
+
+```bash
+cp node_modules//@hpcc-js/wasm/dist/graphvizlib.wasm dist/js/
+```
+
```

Comment:
Suggest moving this to the "Developers" section

Reply:
Planning on solving this before merge, somehow...